test/Parse/multiline_errors.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: %target-typecheck-verify-swift
+
+import Swift
+
+// ===---------- Multiline --------===
+
+_ = """
+    Eleven
+  Mu
+    """ // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
+_ = """
+	Twelve
+\tNu
+	""" // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
+_ = """
+  Thirteen
+	Xi
+  """ // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
+_ = """
+    Fourteen
+  	Pi
+    """ // expected-error@-1{{invalid mix of multi-line string literal indentation}}
+
+_ = """Fourteen
+    Pi
+    """ // expected-error@-2{{invalid start of multi-line string literal}}
+
+_ = """
+    Fourteen
+    Pi""" // expected-error@-0{{invalid end of multi-line string literal}}
+
+_ = """""" // expected-error@-0{{invalid start of multi-line string literal}}
+
```

Comment:
Comment for now, and in the future a better diagnostic, to explain why.

Also, is it only *exactly* 6 quotes in a row that's invalid? Then please add non-error forms for the others. If there are other erroneous forms, please add those with error checks.